stackstorm-compose/README.md

```diff
@@ -0,0 +1,306 @@
+# StackStorm in Docker Compose
+
+This docker-compose is provided as a way to allow someone to "get up and running" quickly with StackStorm using Docker. It is not designed to be used in production, but rather a way to test out StackStorm and facilitate pack development.
+
+## TL;DR
+
+```shell
+git clone git@github.com:stackstorm/st2-dockerfiles
+cd st2-docker/stackstorm-compose
```

Comment:
should this be `cd st2-dockerfiles/stackstorm-compose` instead?

Reply:
Yes, good catch. Fixed.